src/components/GalleryGrid.tsx

```diff
@@ -1,60 +1,74 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import GalleryCard from "./GalleryCard";
 import { GalleryCardType } from "./GalleryCard";
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const staticImage: GalleryCardType = {
   title: "The Bedroom",
-  artist: "Vincent van Gogh",
-  location: "Saint-Rémy-de-Provence, 1889",
+  artist_title: "Vincent van Gogh",
+  date_display: "Saint-Rémy-de-Provence, 1889",
+  place_of_origin: "something",
   image:
     "https://upload.wikimedia.org/wikipedia/commons/0/0b/Vincent_van_Gogh_-_The_Bedroom_-_Google_Art_Project.jpg",
-  altText: "Gustav Klimt painting",
+  image_id: "10",
 };
 
 function GalleryGrid() {
-  const [galleries, setGalleries] = useState<GalleryCardType[]>([]);
-  for (let i = 0; i < 10; i++) {
-    galleries.push(staticImage);
-  }
+  const [artworks, setArtworks] = useState<GalleryCardType[]>([]);
+  useEffect(() => {
+    const fetchArtworks = async () => {
+      try {
+        const response = await fetch(
+          "https://api.artic.edu/api/v1/artworks?page=1&fields=title,artist_title,place_of_origin,date_display,image_id"
+        );
+
+        if (!response.ok) {
+          throw new Error(`Error fetching API. Status: ${response.status}`);
+        }
+        const data = await response.json();
+        setArtworks(data.data);
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    (async () => await fetchArtworks())();
+  }, []);
+
+  const createImageURL = (imageId: string) => {
+    const url = `https://www.artic.edu/iiif/2/${imageId}/full/843,/0/default.jpg`;
+    return url;
+  };
+
+  const generateLocation = (place: string, date: string) => {
+    return `${place}, ${date}`;
+  };
 
   return (
     <>
       <h1>Overview</h1>
       <ul>
         <div className="grid-container">
           {" "}
```

Comment:
What's the purpose of the blank space here?

Reply:
IDK, it gets created when I save the file because of prettier, I'll try to have a look if I can disable this particular formatting